### PR TITLE
Implemented accept attribute support for dropzone.

### LINKED ISSUE #57: Accept Not working on file drop

### PR DESCRIPTION
**Pull request by okonet**
Introduced 2 new callbacks (`onDropAccepted` and `onDropRejected`).
Added `rejectClassName` prop to allow custom styling when drop will be rejected.
Closes #57

### CODE REVIEW

**Comment by paramaggarwal** [comment]
This is great, thanks!

File: index.js [hunk 1]
```diff
@@ -24,16 +27,26 @@ var Dropzone = React.createClass({
...
     activeClassName: React.PropTypes.string,
+    rejectClassName: React.PropTypes.string,
 
     disableClick: React.PropTypes.bool,
     multiple: React.PropTypes.bool,
+    accept: React.PropTypes.string,
+  },
+
+  allFilesAccepted: function(files) {
+    return files.every(file => accept(file, this.props.accept))
```
Comment (paramaggarwal) [inline index.js]: is `.every` available on all browsers? I have always used `.map` instead.

Comment (okonet) [inline index.js]: Don't you use babel to compile the source? https://github.com/paramaggarwal/react-dropzone/blob/master/package.json#L7

The only thing is you'll probably need to add .babelrc with `"optional": ["runtime"],`

Comment (okonet) [inline index.js]: But anyway https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

Comment (okonet) [inline index.js]: BTW I just realized this will not work because of arrow function. Do you mind using ES6 in your project? I could submit another PR with the complete solution.

Comment (paramaggarwal) [inline index.js]: There is already a `babel` pipeline, so we can use anything.

Comment (okonet) [inline index.js]: Right, but not sure it will be transformed with current settings. Let me re-check and send a new PR.

**Comment by quietcodes** [comment]
If rejectClassName is not defined, it is adding an "undefined" class, which seems like an oversight.

**Comment by okonet** [comment]
Thanks. Seems like an issue to me. Mind filing it and I'll take a look soon.
